packages/www/pages/dashboard/streams/[id]/health.tsx

```diff
@@ -1,23 +1,27 @@
 import React, { useCallback } from "react";
 import { Box, Heading, Text, Flex } from "@livepeer.com/design-system";
-import { useApi } from "hooks";
+import { useApi, useAnalyzer } from "hooks";
 import { useEffect, useState } from "react";
 import { StreamInfo } from "hooks/use-api";
 import Chart from "components/Dashboard/Chart";
 import StreamDetail from "layouts/streamDetail";
 import { useRouter } from "next/router";
 import { Stream } from "@livepeer.com/api";
 import { useQuery, useQueryClient } from "react-query";
+import HealthChecksTable from "components/Dashboard/HealthChecksTable";
+import Logger from "@components/Dashboard/Logger";
 
 const interval = 10000;
 const maxItems = 6;
 
 const Health = () => {
-  const [dataChart, setDataChart] = useState<{ name: number; kbps: number }[]>([
-    { name: 0, kbps: 0 },
+  const [dataChart, setDataChart] = useState<{ name: number; kpbs: number }[]>([
+    { name: 0, kpbs: 0 },
```

Comment:
Wasn't it correct like `kbps` "kilobits per second"?

Reply:
oops yeah I was experimenting with something and changed it back with a typo.